File: SecondoAnno/Programmazione_II/Python/esame_11_04_2019/trainer.py
from dipendente import Dipendente

class Trainer(Dipendente):
    ore_settimanali: int
    specialita: str

    def __init__(self, codice: int, nome: str, costo_orario: float, ore_settimanali: int, specialita: str) -> None:
        super().__init__(codice, nome, costo_orario)
        self.ore_settimanali = ore_settimanali
        self.specialita = specialita

    def __str__(self) -> str:
        return super().__str__() + "trainer, " + str(self.ore_settimanali) + " , " + self.specialita + " , - , - , -, " + str(self.costo_orario)

File: SecondoAnno/Programmazione_II/Java/ESAMI/GestionePalestra/Java/Trainer.java
public class Trainer extends Dipendente{

    private int oreSettimanali;
    private double costoOrario;
    private String specialita;
    
    public Trainer(int codice, String nomeDipendente, int oreSettimanali, double costoOrario, String specialita) {
        super(codice, nomeDipendente, costoOrario);
        this.oreSettimanali = oreSettimanali;
        this.costoOrario = costoOrario;
        this.specialita = specialita;
    }

    public String toString() {
        return super.toString() + "\t" + "Trainer" + "\t" + oreSettimanali + "\t" + specialita + "\t-\t-\t-\t" + costoOrario;
    }
    
}
